kubectl-minio/cmd/constants/constants.go

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2020, MinIO, Inc.
+ *
+ * This code is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License, version 3,
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>
+ *
+ */
+
+package constants
+
+import (
+	corev1 "k8s.io/api/core/v1"
+)
+
+// StaticYamlPath is path to the static yaml files
+const StaticYamlPath = "https://raw.githubusercontent.com/nitisht/kubectl-minio/master/cmd/static/static.yaml"
```

Comment:
not nice :-)

Reply:
Yeah agree, I want to add this yaml to the binary, looking at https://github.com/markbates/pkger

Reply:
Yeah agree, I want to add this yaml to the binary, looking at https://github.com/markbates/pkger

Reply:
fixed